src/core/errors/errors.test.ts

```diff
@@ -1,6 +1,26 @@
-import { SDKExecutionError, UnexpectedError } from './errors';
+import { ErrorBase, SDKExecutionError, UnexpectedError } from './errors';
 
 describe('errors', () => {
+  describe('ErrorBase', () => {
+    const error = new ErrorBase('MyKind', 'My message');
+
+    it('has has kind', () => {
```

Comment:
has has has :D

Reply:
🫠 ouch, I didn’t read it after myself yet.